source/faucet/main.d

```diff
@@ -274,8 +274,8 @@ public class Faucet : FaucetAPI
 
     void send ()
     {
-        if (!this.state.update(this.client, Height(this.state.known + 1)))
-            return;
+        if (this.state.update(this.client, Height(this.state.known + 1)))
+            logTrace("State has been updated: %s", this.state.known);
 
         if (this.state.known < 1)
             return logInfo("Waiting for setup to be completed");
```

Comment:
Should the `return` here be removed as well?
I've seen it get stuck here.

Reply:
Looking into it right now.